modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -0,0 +1,100 @@
+* You have installed the OpenShift CLI (`oc`).
+* You have configured remote write storage.
+
+.Procedure
+
+Follow these steps to configure cluster ID labels for metrics for default platform monitoring in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+
+[NOTE]
+====
+If you configure cluster ID labels for metrics for the Prometheus instance that monitors user-defined projects, make similar edits to the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace. 
+Note that the Prometheus config map component is called `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object and not `prometheusK8s`, as it is in the `cluster-monitoring-config` `ConfigMap` object. 
+====
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. In the `writeRelabelConfigs:` section under `data/config.yaml/prometheusK8s/remoteWrite`, add cluster ID label configuration values:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        <endpoint_authentication_credentials>
+        <write_relabel_configs> <1>
+          - <label_config> <2>
+----
+<1> The write relabel configuration settings.
+<2> The source and target label configuration settings for the metrics sent to remote write storage.
++
+For `<write_relabel_configs>`, substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
++
+For `<label_config>`, substitute the label configuration for the metrics sent to the remote endpoint.
++
+The following sample shows how to forward a single metric called `my_metric` with the cluster ID label `cluster_id_1`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        writeRelabelConfigs:
+        - sourceLabels: [__name__]
+          regex: 'my_metric'
+          action: keep
+        - sourceLabels:
+            - __tmp_openshift_cluster_id__ <1>
+          targetLabel: cluster_id_1 <2>
+          action: replace <3>
+----
+<1> The name of a temporary source label for the cluster ID label. This label must begin with [x-]`__tmp_openshift_` followed by the cluster ID label you want to add followed by [x-]`__` as indicated in the sample.
+<2> The name of the cluster ID label added to metrics sent to remote write storage. The first part of this value must match the temporary value followed by a unique identifier for this cluster.
```

Comment:
Like outlined above user are completely free to choose this label name. It might be worth adding some notes about `relabelConfigs` in general:
- If a user chooses a label name that already exists on a metric, the value is overwritten with the cluster id.
- A user should not attempt to use `__tmp_openshift_cluster_id__` as their chosen label name, as this label will be removed as the final relabeling step.

Reply:
Revised per your suggestion